src/Evolution/Systems/Cce/LinearSolve.hpp

```diff
@@ -0,0 +1,328 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/ComplexDataVector.hpp"
+#include "DataStructures/Matrix.hpp"
+#include "DataStructures/Tags.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/Mesh.hpp"
+#include "Evolution/Systems/Cce/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/ApplyMatrices.hpp"
+#include "NumericalAlgorithms/LinearOperators/IndefiniteIntegral.hpp"
+#include "NumericalAlgorithms/LinearOperators/Transpose.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "NumericalAlgorithms/Spectral/SwshTransform.hpp"
+#include "Utilities/Blas.hpp"
+
+extern "C" {
+extern void dgesv_(int*, int*, double*, int*, int*, double*, int*, int*);
```

Comment:
Did we decide on the way we want to wrap this? I suggest to add a wrapper to `NumericalAlgorithms/LinearSolver/`. The parallel linear solvers that the directory currently holds will move to `ParallelAlgorithms` as soon as I find the time to do that.